src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -864,7 +864,7 @@ namespace winrt::TerminalApp::implementation
             }
         });
 
-        events.taskbarToken = control.SetTaskbarProgress([dispatcher, weakThis](auto&&, auto&&) -> winrt::fire_and_forget {
+        events.taskbarToken = control.SetTaskbarProgress([dispatcher, weakThis](auto&&, auto &&) -> winrt::fire_and_forget {

```

Comment:
That shouldn't be happening anymore... Is this formatted with clang-format 13 (ships with VS 17.2 / the one used via `Invoke-CodeFormat`)?